tests/io/aims/test_sets/test_static_generator.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+
+import gzip
+import json
+from glob import glob
+from pathlib import Path
+
+from pymatgen.core import Lattice, Molecule, Structure
+from pymatgen.io.aims.sets.core import StaticSetGenerator
+
+
+def compare_files(test_name, work_dir, ref_dir):
+    for file in glob(f"{work_dir / test_name}/*in"):
+        with open(file) as test_file:
+            test_lines = [line.strip() for line in test_file.readlines()[4:] if len(line.strip()) > 0]
+
+        with gzip.open(f"{ref_dir / test_name / Path(file).name}.gz", "rt") as ref_file:
+            ref_lines = [line.strip() for line in ref_file.readlines()[4:] if len(line.strip()) > 0]
+
+        assert test_lines == ref_lines
+
+    with open(f"{ref_dir / test_name}/parameters.json") as ref_file:
+        ref = json.load(ref_file)
+    ref.pop("species_dir", None)
+
+    with open(f"{work_dir / test_name}/parameters.json") as check_file:
+        check = json.load(check_file)
+    check.pop("species_dir", None)
+
+    assert ref == check
```

Comment:
there's some redundancy between the test files in this directory. they all define nearly or even fully identical `compare_files` functions. can we put this function into an `__init__.py` file and import `compare_files` and other helpers from there?

Reply:
we'll figure it out. happy to help if errors come up :)

Reply:
Now I remember what the error is, one of the packages installed using `pip install .[strict]` creates a `tests` directory in `site-packages` which conflicts with the `tests/__init__.py` file. I am in the process of seeing if I can figure out which package it is.

Reply:
The offending package is `pybtex==0.24.0`

Reply:
I'm not sure how to override this without some sort of very hacky workaround with the sys.path

Reply:
you could try [this SO answer](https://stackoverflow.com/a/46976704), or turn the helper function into a `pytest` fixture in wich case there's no need to import it. it'll be available as a function argument to each test function.

Reply:
Okay I will try the pytest fixture route first